src/main/java/new_metrics/CustomGammaDistribution.java

```diff
@@ -24,12 +9,12 @@
 import org.apache.commons.math3.util.FastMath;
 
 /**
- * Implementation of the Gamma distribution.
+ * Implementation of the Gamma distribution. Customized from org.apache.commons.commons-math3 GammaDistribution.
  *
  * @see <a href="http://en.wikipedia.org/wiki/Gamma_distribution">Gamma distribution (Wikipedia)</a>
  * @see <a href="http://mathworld.wolfram.com/GammaDistribution.html">Gamma distribution (MathWorld)</a>
  */
-public class GammaDistribution extends AbstractRealDistribution {
```

Comment:
I think we could eventually compose a `CustomGammaDistribution` object with a `GammaDistribution` object so as to delegate as much logic from the former to the latter and not have to maintain a separate copy. But that can be done in a future PR.